<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogsWidget</class>
 <widget class="QWidget" name="LogsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>953</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,100">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item row="5" column="1">
          <widget class="QDateTimeEdit" name="maxDateTime">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="showGroupSeparator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="maximumDate">
            <date>
             <year>7930</year>
             <month>12</month>
             <day>31</day>
            </date>
           </property>
           <property name="minimumDate">
            <date>
             <year>1780</year>
             <month>9</month>
             <day>14</day>
            </date>
           </property>
           <property name="displayFormat">
            <string>dd-MM-yyyy HH:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="dateTimePreset">
           <item>
            <property name="text">
             <string>Today</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yesterday</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>This Week</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last Week</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>This Month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last Month</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>From</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDateTimeEdit" name="minDateTime">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="showGroupSeparator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="maximumDate">
            <date>
             <year>7930</year>
             <month>12</month>
             <day>31</day>
            </date>
           </property>
           <property name="minimumDate">
            <date>
             <year>1780</year>
             <month>9</month>
             <day>14</day>
            </date>
           </property>
           <property name="displayFormat">
            <string>dd-MM-yyyy HH:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>To</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="useDateTimePreset">
           <property name="text">
            <string>Preset</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QCheckBox" name="warning">
           <property name="text">
            <string>Warning</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="error">
           <property name="text">
            <string>Error</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="info">
           <property name="text">
            <string>Info</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="verbose">
           <property name="text">
            <string>Verbose</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="exportLogs">
           <property name="text">
            <string>Export Logs</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/export.png</normaloff>:/icons/ui/export.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeView" name="list">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>useDateTimePreset</tabstop>
  <tabstop>dateTimePreset</tabstop>
  <tabstop>maxDateTime</tabstop>
  <tabstop>minDateTime</tabstop>
  <tabstop>verbose</tabstop>
  <tabstop>info</tabstop>
  <tabstop>warning</tabstop>
  <tabstop>error</tabstop>
  <tabstop>list</tabstop>
 </tabstops>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>useDateTimePreset</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateTimePreset</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDateTimePreset</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDateTimePreset</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDateTimePreset</sender>
   <signal>toggled(bool)</signal>
   <receiver>minDateTime</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>12</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDateTimePreset</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxDateTime</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
